DirectProgramming/DPC++FPGA/ReferenceDesigns/mvdr_beamforming/README.md

```diff
@@ -204,6 +205,7 @@ PASSED
 |`StreamingQRD.hpp`              | StreamingQRD kernel, performs Q-R Decompostion on a matrix
 |`Transpose.hpp`                 | Transpose kernel, reorders data for the StreamingQRD kernel
 |`Tuple.hpp`                     | A templated tuple that defines the NTuple class which is used for pipe interfaces
+|`UDP.hpp`                       | This code is **only** relevant for using the real IO pipes (i.e. not in the devcloud). This is discussed later in the [Using Real IO-pipes Section](#using-real-io-pipes).
```

Comment:
Other files have changed (some added some deleted), can you update them?

Did you intentionally not add the files associated with the loopback test?  Do you not think they should be listed here?

Reply:
Which have changed? I can add the loopback test one.

Reply:
|`InputDemux.hpp`           | InputDemux kernel, separates training and processing data.

remove: NullPipe.hpp

|`ParallelCopyArray.hpp`   | Defines the ParallelCopyArray class, an array that supports unrolled copy / assign operations
|`PipeDuplicator.hpp`        | Defines the PipeDuplicator class, creates multiple copies of a pipe for fan-out.
udp_loopback_test.cpp (you seem to have forgotten to change the name from mvdr_beamforming.cpp)